server/methods/accounts/accounts.js

```diff
@@ -396,7 +396,7 @@ export function addressBookAdd(address, accountUserId) {
     accountsUpdateQuery.$set.name = address.fullName;
   }
 
-  Meteor.users.update(Meteor.userId(), userUpdateQuery);
+  Meteor.users.update(userId, userUpdateQuery);
```

Comment:
I'm not entirely sure why we have this information _both_ in `Accounts` and in `users`, so if this is incorrect, please let me know!